src/components/SelectCol.js

```diff
@@ -17,27 +21,13 @@ class SelectCol extends React.Component {
         }
     }
 
-    //handle Event functions
-    openForm() {
-        const x = document.getElementById( "language-options" );
-        if ( x.style.display === "none" ) {
-            x.style.display = "block";
-        } else {
-            x.style.display = "none";
-        }
-
-        document.addEventListener( 'click', function() {
-            x.style.display = "none";
-        } );
-    }
-
-    handleClick( e ) { //stop propagation of the react
-        e.stopPropagation();
-        e.nativeEvent.stopImmediatePropagation();
```

Comment:
one original functionality is missing.
Before, after clicking the selectComponent to see all the list, users can close the click by clicking anywhere on the website. I think some of the deleted code here (line 20 - 36) is doig that thing. So just have a look and see if you can get that functionality back?